examples/simple-lambda-worker/lib/simple-lambda-worker-stack.ts

```diff
@@ -48,16 +59,24 @@ export class SimpleLambdaWorkerStack extends cdk.Stack {
         lambdaProps: {
           environment: {
             EXAMPLE_ENV_VAR: "example value",
+            SUCCESS_QUEUE_URL: successQueue.queueUrl,
           },
           entry: "src/lambda/simple-worker.js",
           handler: "simpleLambdaWorker",
           memorySize: 1024,
-          timeout: cdk.Duration.minutes(5),
+          timeout: cdk.Duration.seconds(30),
           vpc: vpc,
           vpcSubnets: { subnetType: ec2.SubnetType.PRIVATE },
+          policyStatements: [
+            new iam.PolicyStatement({
+              effect: iam.Effect.ALLOW,
+              actions: ["sqs:SendMessage"],
+              resources: [successQueue.queueArn],
+            }),
+          ],
         },
         queueProps: {
-          maxReceiveCount: 3,
+          maxReceiveCount: 1,
```

Comment:
Reduce the times a failed message is retried. Again - this is to reduce the time taken for the message to appear on the DLQ in the integration test.